pulumi/README.md

```diff
@@ -0,0 +1,5 @@
+# Pulumi example
+
+1. pulumi new python
+2. pulumi config set --secret apiKey <YOUR_API_KEY>
+3. pulumi up
```

Comment:
Add a step of copying the code or something

Reply:
This README file located under pulumi folder ...